ansible_base/resource_registry/models/service_identifier.py

```diff
@@ -23,5 +23,11 @@ def save(self, *args, **kwargs):
 def service_id():
     global _service_id
     if not _service_id:
-        _service_id = str(ServiceID.objects.first().pk)
+        service_obj = ServiceID.objects.first()
+        if service_obj:
+            _service_id = str(service_obj.pk)
+        else:
+            # Create a ServiceID if none exists
+            service_obj = ServiceID.objects.create()
```

Comment:
Oh, this is a major thing I had on my agenda.

https://issues.redhat.com/browse/AAP-51352

I had assumed we couldn't do it like this. But I'm not saying no.